### DISCUSSION TITLE
Question about B: drive conflict.

### DISCUSSION THREAD

**Discussion by IgnusVam** [KoboldAI/KoboldAI-Client#450] [Q&A] [answered]
My computer has a mounted hard drive already lettered as a B: drive.
Is Kobold AI still safe to use with this hard drive installed, even installing it to not use it, or will it simply search for the B: drive and unmount my hard drive after usage?
Am I able to have K.AI use a different drive letter for the locally run install, or does it only support the B: Drive?

**Comment by henk717** [comment, marked as answer]
It will break the trick, you could use subfolder mode but this repo is to old for that to work reliably.
I recommend our newer product https://koboldai.org/cpp which can run GGUF models. That one is standalone and doesn't rely on the B: drive trick to be shipped.
